recipes/freud/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "freud" %}
+{% set version = "0.8.2" %}
+{% set sha256 = "e4c2c3dbddb70f42fb526375186b882ce35073d75ca494cabbec478aa523ab48" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://glotzerlab.engin.umich.edu/Downloads/{{ name }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+
+  host:
+    - python
+    - cython
+    - tbb
+    - boost
+    - numpy
+
+  run:
+    - python
+    - tbb
+    - {{ pin_compatible('numpy') }}
+    - scipy
+
+test:
+  imports:
+    - freud
+
+about:
+  home: https://bitbucket.org/glotzer/{{ name }}
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'The freud library provides users the ability to analyze generic data from a variety of sources including simulation and experimental data for advanced metrics such as the radial distribution function and various order parameters.'
+  doc_url: http://{{ name }}.readthedocs.io/
+  dev_url: https://bitbucket.org/glotzer/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - bdice
+    - vyasr
```

Comment:
@vyasr @joaander 
Are you okay with being a maintainer for this conda-forge recipe?  For details as to what this entails see [the docs](https://conda-forge.org/docs/recipe.html#maintainer-role)

Reply:
Yes.

Reply:
Yes.